docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc

```diff
@@ -75,6 +75,28 @@ To work around this problem, uninstall the {agent} and install it again with
 {fleet-server} enabled during the bootstrap process.
 ====
 
+[[known-issue-2103-8-6-2]]
```

Comment:
We should make this entry and the one for 8.6.0 consistent. I like aspects of both descriptions, so I'll leave it to you to decide. (I'll wait for your update before I suggest edits to the description.)

Reply:
Done, I somehow missed that this was already listed for 8.6.0. It applies to all versions.